client/src/featureform/register.py

```diff
@@ -571,6 +573,67 @@ def register_file(
             self.__registrar, name, owner, self.name(), path, variant, description
         )
 
+    def register_directory(
+        self,
+        name,
+        path,
+        description="",
+        variant: str = "",
+        owner="",
+        tags: List[str] = [],
+        properties: dict = {},
+    ):
+        """Register a directory.
+        When registering a directory, files can be interacted with as a table with columns "filename" and "body".
+        For example:
+        filename                  |            body
+        --------------------------|---------------------------------------
+        featureform_docs.txt      | Featureform allows data scientists....
```

Comment:
I wonder what the df constraints are for a column. If you have a large ass file will that fit in the column. Something to think about

Reply:
This is limited by RAM